### PR TITLE
Updated packer instruction in README

### PR DESCRIPTION
**Pull request by andersevenrud**
Since colorbuddy is a dependency this should be placed inside a "require" parameter for packer.nvim

### CODE REVIEW

**Review by maaslalani — APPROVED** [review]

**Comment by maaslalani** [comment]
Oops forgot you have merge powers! Feel free to merge the other PRs

**Comment by andersevenrud** [comment]
I thought I'd pass it by you guys and await complete approval before merging myself :blush:

**Comment by andersevenrud** [comment]
Oh, and thanks for the :boom:merge powers:boom:. Really appreciate it!

**Pull request merged**
